frontend/src/metabase/home/components/EmbedHomepage/EmbedHomepage.tsx

```diff
@@ -1,9 +1,14 @@
+import { useSetting } from "metabase/common/hooks";
 import { useSelector } from "metabase/lib/redux";
 import { getDocsUrl } from "metabase/selectors/settings";
 
 import { EmbedHomepageView } from "./EmbedHomepageView";
 
 export const EmbedHomepage = () => {
+  const embeddingAutoEnabled = useSetting("setup-embedding-autoenabled");
+  const licenseActiveAtSetup = useSetting("setup-license-active-at-setup");
+  const exampleDashboardId = undefined; // will come from a setting
```

Comment:
this component is not used at the moment, some props are still not connected (this + the plan later on)